docs/QuickStartGuides/3-CreatingAMode.md

```diff
@@ -1,11 +1,42 @@
 # Creating A New Mode
 
-*Note: this guide is only for versions greater than 0.1. If you'd like to check out this functionality sooner, it is on the `refactor` branch.*
+Once you have familiarized yourself with Symbulation and its default mode, you might be interested in designing your own experiment, adding functionality to the codebase, and collecting data. The existing modes include default (Host and Symbiont), efficient (Efficient Host and Efficient Symbiont), lysis (Bacterium and Phage), and public goods game or PGG (PGG Host and PGG Symbiont). There are several steps to creating your own world, including following conventions for file structure, adding your own organisms, designing tests, adding a WorldSetup file, adding targets to the makefile, and more. This guide will walk you through how to properly add all of these features.
 
-Once you have familiarized yourself with Symbulation and its default mode, you might be interested in designing
-your own experiment, adding functionality to the codebase, and collecting data. The existing modes include
-default (Host and Symbiont), efficient (Efficient Host and Efficient Symbiont), lysis (Bacterium and Phage),
-and public goods game or PGG (PGG Host and PGG Symbiont). There are several steps to creating your own world,
-including following conventions for file structure, adding your own organisms, designing tests, adding a 
-WorldSetup file, adding targets to the makefile, and more. 
+First, you must decide if your new mode falls under one of two scenarios. In the first scenario, you wish to change the processes of an existing organism, but will not be adding any new traits. The second scenario includes the addition of new traits, or genome values, and will therefore also require functions that track the evolution of this new trait. Depending on the goals of your project and which scenario it falls under (1 or 2), you will need to add to the codebase in a varying manner. Most sections below pertain to both scenarios and should be completed no matter what. Sections that are specific to a particular scenario will be labeled accordingly.
 
+## Primary Folder
+Choose a one word descriptor for your new mode. Then make a folder in `source` named `<name>_mode`. Inside of this folder will be a world setup source file, as well as any necessary header files.
+
+### Organisms
+Next, you will need to add new header files for your new organisms. The host organism(s) must extend the `Host.h` class in the `source/default_mode`, and similarly, the symbiont organism(s) must extend the `Symbiont.h` class.
```

Comment:
Would be good to include a code snippet on how you extend a class, probably with a class called 'ExampleHost' or something